proxies/live/apiproxy/proxies/default.xml

```diff
@@ -558,11 +565,17 @@
                 <Step>
                     <Name>Javascript.SetDefaultAccessAndRefreshTokenTimes</Name>
                 </Step>
-                <!-- Next step allows optional REDUCTION of access_token_expiry KVM values
-            using _access_token_expiry_ms form param -->
+                <!-- Using JavaScript to evaluate the requested override time, due to apigee conditional failing to evaluate properly -->
+                <Step>
+                  <Name>AssignMessage.SetAccessTokenExpiry</Name>
```

Comment:
don't need (and shouldn't use) this assign message here. 
Javascript.SetDefaultAccessAndRefreshTokenTimes is setting up the apigee.access_token_expiry_ms variable